### PR TITLE
Ini file configuration save

### PR DESCRIPTION
**Pull request by gloomy4eyes**
good afternoon. 
I added an implementation to save ini configuration files & test for it.

### CODE REVIEW

File: Util/include/Poco/Util/IniFileConfiguration.h [hunk 1]
```diff
@@ -69,6 +69,14 @@ class Util_API IniFileConfiguration: public AbstractConfiguration
 		/// Loads the configuration data from the given file, which
 		/// must be in initialization file format.
 
+  void save(std::ostream& ostr);
```
Comment (aleks-f) [inline Util/include/Poco/Util/IniFileConfiguration.h]: tabs, please. See [Coding Style](http://www.appinf.com/download/CppCodingStyleGuide.pdf)

File: Util/testsuite/src/IniFileConfigurationTest.cpp [hunk 2]
```diff
@@ -89,6 +89,61 @@ void IniFileConfigurationTest::testLoad()
 	assertTrue (std::find(keys.begin(), keys.end(), "Prop1") == keys.end());
 }
 
+void IniFileConfigurationTest::testSave() {
```
Comment (aleks-f) [inline Util/testsuite/src/IniFileConfigurationTest.cpp]: coding style

**Comment by gloomy4eyes** [comment]
please, check my commits, all tests have been passed

**Comment by ark0f** [comment]
Please, be patient. Reviewers check PRs in their free time.

**Pull request closed without merging**